lib/src/quiz/quiz_view_model.dart

```diff
@@ -34,15 +34,17 @@ class QuizViewModel extends FutureViewModel {
 
   @override
   Future futureToRun() async {
-    final kana = await _kanaRepository
```

Comment:
Why switch to a `.then` ?

Reply:
C'était pour tenter de régler un problème car le composant disait qu'il était prêt alors que les questions n'étaient pas prêtes. Du coup j'avais testé de retourner un `Future` (tel que c'est censé faire) pour voir si ça réglait mon problème et c'était le cas.
J'ai retesté sans et ça roule, ça devait être lié au soucis de chargement d'isar.

Reply:
Très probablement, en soit on retourne un Future<void>, si tu voulais retourner le set de question faudrait changer le view-model pour un `FutureViewModel<List<Questions>>`